keywords/MobileRestClient.py

```diff
@@ -388,15 +404,18 @@ def create_user(self, url, db, name, password, channels=None, roles=None, auth=N
         if roles is None:
             roles = []
 
-        types.verify_is_list(channels)
-        types.verify_is_list(roles)
-
         data = {
             "name": name,
             "password": password,
-            "admin_channels": channels,
```

Comment:
It is possible to have both a top level `admin_channels` and those under `collection_access` for backwards compatibility; the top level maps to the `_default._default` collection